src/v2/guide/single-file-components.md

```diff
@@ -1,48 +1,48 @@
 
-Todos estos puntos estan resueltos por **single-file components** con extensión `.vue`, lo cual es posible con herramientas como Webpack or Browserify.
+Todo esto se resuelve mediante componentes de un solo archivo con extensión .vue, hecho posible con herramientas de compilación como Webpack o Browserify.
 
 Aquí tenemos un ejemplo de un archivo que llamaremos `Hello.vue`:
 
-<a href="https://gist.github.com/chrisvfritz/e2b6a6110e0829d78fa4aedf7cf6b235" target="_blank"><img src="/images/vue-component.png" alt="Single-file component example (click for code as text)" style="display: block; margin: 30px auto;"></a>
+<a href="https://gist.github.com/chrisvfritz/e2b6a6110e0829d78fa4aedf7cf6b235" target="_blank"><img src="/images/vue-component.png" alt="Ejemplo de Componente de un Solo Archivo (click para ver código como texto)" style="display: block; margin: 30px auto;"></a>
 
 Ahora tenemos:
 
 - [Resaltado completo de la sintaxis](https://github.com/vuejs/awesome-vue#source-code-editing)
-- [Módulos CommonJSCommonJS modules](https://webpack.js.org/concepts/modules/#what-is-a-webpack-module)
-- [Definición de alcance del CSS por Component](https://vue-loader.vuejs.org/en/features/scoped-css.html)
+- [Módulos CommonJS](https://webpack.js.org/concepts/modules/#what-is-a-webpack-module)
+- [CSS con ámbito de componente](https://vue-loader.vuejs.org/en/features/scoped-css.html)
 
-Como prometimos, podemos también usar preprocesadores como Pug, Babel (con módulos ES2015), y Stylus para componentes más limpios y con más características.
+Como prometimos, podemos también usar preprocesadores como Pug, Babel (con módulos ES2015) y Stylus para obtener componentes más limpios y con más funcionalidades.
 
-<a href="https://gist.github.com/chrisvfritz/1c9f2daea9bc078dcb47e9a82e5f7587" target="_blank"><img src="/images/vue-component-with-preprocessors.png" alt="Single-file component example with preprocessors (click for code as text)" style="display: block; margin: 30px auto;"></a>
+<a href="https://gist.github.com/chrisvfritz/1c9f2daea9bc078dcb47e9a82e5f7587" target="_blank"><img src="/images/vue-component-with-preprocessors.png" alt="Ejemplo de Componente de un Solo Archivo con preprocesadores (click para ver código como texto)" style="display: block; margin: 30px auto;"></a>
 
-Estos lenguajes específicos son solo ejemplos. Usted podrías usar facilmente Bublé, TypeScript, SCSS, PostCSS - o cualquier otro preprocesador que le ayude a ser productivo. Si utiliza Webpack con `vue-loader`, esto también tiene soporte de primera clase para módulos CSS.
+Estos lenguajes específicos son solo ejemplos. Podrías usar, con la misma facilidad, Bublé, TypeScript, SCSS, PostCS, o cualquier otro preprocesador que te ayude a ser productivo. Si utilizas Webpack con `vue-loader`, también te ofrece soporte de primera clase para módulos CSS.
```

Comment:
- Small typo (missing `S` on `PostCSS`)
- Some more recommendations (that I think are more the kind of language we are using)

```suggestion
Estos lenguajes específicos son solo ejemplos. Podrías usar, con la misma facilidad, Bublé, TypeScript, SCSS, PostCSS, o cualquier otro pre-procesador que le ayude a ser productivo. Si utiliza Webpack con `vue-loader`, también le ofrece soporte de primera clase para módulos CSS.
```

Reply:
I don't know what treatment we're using in general, but I do think it needs to be consistent. Are we using formal ("usted") or casual ("tú") addressing? The rest of the translation is using "tú", let me know if I should change it all.

Reply:
we are using formal addressing ("usted"). If we found casual addressing we should change it.

Reply:
Changed to formal treatment.